src/components/CVoxel/VoxelDetail/VoxelDetail.tsx

```diff
@@ -270,17 +292,23 @@ export const VoxelDetail: FC<Props> = ({
             </div>
           </div>
         )}
-
-        {updatable && isOwner && (
-          <div className="text-right">
-            <Button
-              text="Update"
-              color="primary"
-              buttonType="button"
-              onClick={handleUpdate}
-            />
-          </div>
-        )}
+        <div className="w-full flex items-center justify-end space-x-1 pt-7">
+          {(isSemiCRDL && offchainItem && offchainItem.id) && (
+            <div className="text-right">
+              <CopyRequestURLButton id={offchainItem.id} />
+            </div>
+          )}
+          {updatable && isOwner && (
```

Comment:
By the way, if you don't refresh the screen after the tx is created, the form will not be updated from the state where it can be filled in.

Reply:
> [IMO] I can't update the voxel I created in preview environment(the update button disappeared), maybe it's because of the updatable condition.
=> Currently, only updating the signature is available, so the update button is shown only when your partner(payee/payer) sign your sig request :)

Reply:
>By the way, if you don't refresh the screen after the tx is created, the form will not be updated from the state where it can be filled in.

Thanks for reporting ! fixed that :)

Reply:
> Currently, only updating the signature is available, so the update button is shown only when your partner(payee/payer) sign your sig request :)

I see!